### ISSUE TITLE
Update Example Python Scripts to Python3 or Specify Python2 Requirement in Docs

### ISSUE THREAD

**Issue by TomMoeras** [hpcugent/vsc_user_docs#588]
While browsing through the example scripts in the docs, I noticed that the Python scripts seem to utilize Python 2 print statements. This could potentially lead to errors when running these scripts in a Python 3 environment. So not really a big issue and something that can be easily resolved.

One of the scripts uses the following structure:

```python
# /examples/Program-examples/01_Python/primes.py
#! /usr/bin/env python
import datetime
import time
import sys

def print_time(str):
    now = time.time()
    st = datetime.datetime.fromtimestamp(now).strftime('%Y-%m-%d %H:%M:%S')
    print str, st
    return int(now)

def ask_int(prompt, retries=3):
    while True:
        num = int(input(prompt))
        if num > 1:
            return num
        else:
            print "Please enter an integer which is bigger than one."
            retries -= 1
        if retries <= 0:
            print('Too many attempts, try again later.')
            sys.exit()
```
The `print` statements in this script are written in the Python 2 style, without parentheses, which would lead to syntax errors in Python 3. 

Possible fixes are:
1. **Update the Scripts:** The scripts could be updated to Python 3. This would involve minor changes, such as modifying print statements to use parentheses.
2. **Specify Python Version in Documentation:** Alternatively, if maintaining the scripts in Python 2 is preferable, it would be beneficial to clearly specify in the documentation that these scripts are intended to be run with Python 2.

**Comment by boegel** [comment]
@laraPPr Is this fixed with #592?

**Comment by laraPPr** [comment]
Yes, those were all updated.
